edge/pkg/devicetwin/dtmanager/device_test.go

```diff
@@ -102,7 +102,7 @@ func TestDeviceStartAction(t *testing.T) {
 	}
 }
 
-// TestDevicetHeartBeat is function to test Start() when value is passed in HeartBeatChan.
+// TestDeviceHeartBeat is function to test Start() when value is passed in HeartBeatChan.
```

Comment:
TestDeviceStartHeartBeat?

Reply:
Thanks for your review. 

I thought the original name is better. The function Start/Run is the entry for almost all processes. It's a little weird to add this word.

Reply:
Sounds good, so the func name in L106 need to be fixed?

Reply:
Yes, you're right. Refined it.

Thanks.